cluster/config-defaults.yaml

```diff
@@ -208,8 +208,8 @@ cluster_dns: "coredns"
 coredns_log_svc_names: "true"
 coredns_max_upstream_concurrency: 0 #0 means there is not concurrency limits
 
-kuberuntu_image_v1_16: {{ amiID "zalando-ubuntu-kubernetes-production-v1.16.8-master-96" "861068367966" }}
-kuberuntu_image_v1_17: {{ amiID "zalando-ubuntu-kubernetes-production-v1.17.4-master-98" "861068367966" }}
+kuberuntu_image_v1_17: {{ amiID "zalando-ubuntu-kubernetes-production-v1.17.5-master-103" "861068367966" }}
+kuberuntu_image_v1_18: {{ amiID "zalando-ubuntu-kubernetes-production-v1.18.2-master-103" "861068367966" }}
```

Comment:
Let's add this in a dedicated v1.18 PR otherwise it will just be a source of potential merge conflicts :)

Reply:
:+1: https://github.com/zalando-incubator/kubernetes-on-aws/pull/3228